docs/UsingCppInterOp.rst

```diff
@@ -67,33 +67,204 @@ that it was designed for.
 
 Further Enhancing the Dynamic/Automatic bindings in CPPYY
 =========================================================
+
+CppInterOp ties into the following major enhancements in cppyy:
+
+-  **Reducing dependencies** of cppyy to minimize code bloat and make it
+   faster.
+
+-  A **cppyy-Style Library** (i.e., CppInterOp) that enables interoperability
+   with C++ code, bringing the speed and efficiency of C++ to simpler, more
+   interactive languages like Python.
+
+**Reducing Dependencies**
+
+Recent work done on cppyy has been focused on removing unnecessary
+dependencies on domain-specific infrastructure (e.g., the ROOT
+framework). The idea was to convert the cppyy-backend to use Cling
+directly (instead of ROOT meta), and then use it in cppyy.
+
+Only a small set of APIs are needed to connect to the interpreter, since
+other APIs are already available in the standard compiler. This is what
+led to the creation of CppInterOp (a library of helper functions), that
+helped extract out things that were unnecessary for cppyy, etc.
+
+The API surface is now incomparably smaller and simpler than what it
```

Comment:
its confusing - which "api" surface?

Reply:
This was written as described in info session...
 I'm assuming that this refers to the cppyy API surface (which should be simpler after bloat removal)? Please confirm